### ISSUE TITLE
Table formatting is something wrong

### ISSUE THREAD

**Issue by geffzhang** [lunet-io/markdig#44]
The pandoc pipe table example， Only two columns of the table, analytical results are not expected：
```
| abc | def | 
|---|:---|
| cde| ddd| 
| eee| fff|
| fff | fffff   | 
|gggg  | ffff | 
```
[test on babelmark](https://babelmark.github.io/?text=%7C+abc+%7C+def+%7C+%0A%7C---%7C%3A---%7C%0A%7C+cde%7C+ddd%7C+%0A%7C+eee%7C+fff%7C%0A%7C+fff+%7C+fffff+++%7C+%0A%7Cgggg++%7C+ffff+%7C+)

var pipeline = new MarkdownPipelineBuilder().UseAdvancedExtensions().Build();
var result = Markdown.ToHtml(markdownDocument, pipeline);

version 0.7.4

**Comment by christophano** [comment]
I don't know if it helps, but this table works if you remove the trailing whitespace from the ends of a few of the lines.
I don't know what the spec says about triling whitespace on pipe tables (or indeed, if there is a spec).

**Comment by xoofx** [comment]
@christophano yep, I'm on it.

**Comment by xoofx** [comment]
This should be fixed by commit fa281f1
